gpu_ocean/SWESimulators/CDKLM16.py

```diff
@@ -258,6 +268,13 @@ def __init__(self, \
         )
 
 
+        def subsample(data, factor):
```

Comment:
This may be slightly off - half a cell or so, depending on how we use the textures. However, I think it is of mostly academic interest to do this 100% correctly at this point in time (the coriolis parameter will perhaps be 400 meters off or so)

Reply:
function input "factor" not used in function body. Guess that should be the hard coded 10?

Reply:
Also, when running the NorKyst800_Experiment notebook I get "ValueError: When on a regular grid with x.size = m and y.size = n, if z.ndim == 2, then z must have shape (n, m)" from the interp2d function when subsampling the angle... Does it work for you? I tested with Python 3.6 and scipy 1.3.1

Reply:
This PR is actually missing one commit that fixes that last one...